hardwarecheckout/templates/includes/macros/selection.html

```diff
@@ -0,0 +1,36 @@
+{% macro selection(id, header, message, action, data_id, data_userid, button_class) -%}
+	<div data-id="{{ data_id }}" class="ui {{ id }} modal">
+		<div class="header"> {{ header }} </div>
+		{% if message %}
+			<div class="content"> 
+				<p>{{ message }}</p>
+			</div>
+		{% endif %}
+		<div class="actions">
+			<a class="ui primary approve button {{ button_class }}" href="#" 
```

Comment:
It would be cool if this button had keyboard focus by default, so that you can just hit enter to select, or tab-enter to select the other button. Take a look here - https://www.w3schools.com/tags/att_button_autofocus.asp